### ISSUE TITLE
[Feature] Auto Resolution Switch (like real MSX !!) [sf#61]

### ISSUE THREAD

**Issue by openMSX-import** [openMSX/openMSX#85]
**Reported by *anonymous on 2003-11-03 01:38 UTC**
Automatically Switch to Hi-Resolution (640x480)when 
needed...rather than blended image in Lo Resolution mode
when using SDLLoRenderer....like real MSX !!

**Comment by openMSX-import** [comment]
**Commented by manuelbi on 2003-11-03 17:01 UTC**
Logged In: YES 
user_id=78178

I wonder what would be the use of this. If you want hires,
just choose SDLHi. No need to switch to high res then.
Switching back to 320x240 when that is possible could be an
option to improve performance. But it will only work in full
screen mode (unless you want your window getting 1/4 of
previous size) and that only works when you have the 320x240
mode in your X server. 
In other words: I think I can close this feature request. If
you want it like a real MSX, use a hires renderer like SDLHi
or SDLGL.

**Comment by openMSX-import** [comment]
**Commented by mthuurne on 2003-11-05 01:46 UTC**
Logged In: YES 
user_id=358343

As Manuel says, in windowed mode this would produce the strange 
effect of a growing and shrinking window (unlike real MSX, where the 
TV/monitor displays a fixed size image). 

In full screen mode it could work, but most monitors take a while to 
switch resolutions, so it's far from seamless. For example, on my 
system it takes about 2 seconds to switch from 1280x1024 to 640x480. 

So right now I'm not in favour of this feature. But maybe if you explain 
why you want it, we can find a solution?

**Comment by openMSX-import** [comment]
**Updated by manuelbi on 2003-11-15 18:48 UTC**
- **priority**: 5 --> 1

**Comment by openMSX-import** [comment]
**Commented by manuelbi on 2003-11-26 21:52 UTC**
Logged In: YES 
user_id=78178

Closing since it's not making sense to us and submitter
doesn't add explanation.

**Comment by openMSX-import** [comment]
**Updated by manuelbi on 2003-11-26 21:52 UTC**
- **status**: open --> closed
